test-utils/testlib/Cargo.toml

```diff
@@ -16,6 +16,7 @@ rand = "0.7"
 serde_json = "1"
 futures = "0.3"
 lazy_static = "1.4"
```

Comment:
is `lazy_static` still needed after this change?

Reply:
Yeah, it's used in some places for test globals:

```
lazy_static! {
    static ref HEAVY_TESTS_LOCK: Mutex<()> = Mutex::new(());
}
```